src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/PaymentProviderWorkflow.cs

```diff
@@ -23,66 +29,63 @@ namespace Umbraco.Forms.Integrations.Commerce.EMerchantPay
 {
     public class PaymentProviderWorkflow : WorkflowType
     {
-        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly PaymentProviderSettings _paymentProviderSettings;
 
-        private readonly IUmbracoContextAccessor _umbracoContextAccessor;
+        private readonly IHttpContextAccessor _httpContextAccessor;
 
         private readonly ConsumerService _consumerService;
 
         private readonly PaymentService _paymentService;
 
+        private readonly UrlHelper _urlHelper;
+
         #region WorkflowSettings
 
-        [Setting("Usage",
+        [Core.Attributes.Setting("Usage",
             Description = "Payment usage description",
             View = "TextField")]
         public string Usage { get; set; }
 
-        [Setting("Amount",
+        [Core.Attributes.Setting("Amount",
             Description = "Payment amount (without decimals)",
             View = "TextField")]
         public string Amount { get; set; }
 
-        [Setting("Currency",
+        [Core.Attributes.Setting("Currency",
             Description = "Payment currency",
-            PreValues = "DKK,EUR,USD",
-            View = "Dropdownlist")]
+            View = "~/App_Plugins/UmbracoForms.Integrations/Commerce/eMerchantPay/currency.html")]
         public string Currency { get; set; }
 
-        [Setting("Supplier",
+        [Core.Attributes.Setting("Supplier",
```

Comment:
Wondering if this, and perhaps `Usage` could be something we add to the configuration file, so the editor doesn't need to set for each instance?  Not sure, but it sounds like it would be something the same for all payments taken from a single website.

Reply:
Moved Supplier and Usage to configuration file.